scripts/tags/exturl.js

```diff
@@ -0,0 +1,57 @@
+/* global hexo */
+// Usage: {% exturl text url "title" %}
+// Alias: {% extlink text url "title" %}
+
+'use strict';
+
+var util = require('hexo-util');
```

Comment:
@iissnan ok, ckeck this: https://github.com/hexojs/hexo-util

I don't remember for now, but possibly i was install it together with hexo.
Directory `hexo-util` have same install data with directory `hexo`. But, in standart plugins (https://hexo.io/plugins/) `hexo-util` not present.

Anyway, if this issue present, need to reccomend peoples install this utilities?
```
$ npm install hexo-util --save
```